packages/wine.rb

```diff
@@ -103,5 +110,18 @@ def self.remove
       end
       puts
     end
+    if Dir.exist? "#{@xdg_config_home}/.wine"
+      puts
+      print "Would you like to remove #{@xdg_config_home}/.wine? [y/N] "
+      response = $stdin.getc
+      case response
+      when 'y', 'Y'
+        FileUtils.rm_rf "#{@xdg_config_home}/.wine"
+        puts "#{@xdg_config_home}/.wine removed.".lightred
+      else
+        puts "#{@xdg_config_home}/.wine saved.".lightgreen
+      end
+      puts
+    end
```

Comment:
Don't we want to remove the prompt above this?

Reply:
That should only show up if a `~/.wine` folder exists, right?

Reply:
What is the difference between `#{HOME}/.wine` and `#{@xdg_config_home}/.wine`?

Reply:
Why not do this?
```
    config_dirs = ["#{HOME}/.wine", "#{@xdg_config_home}/.wine"]
    config_dirs.each { |config_dir|                                                                                                                                                           
      if Dir.exists? config_dir                                                                                                                                                               
         print "\nWould you like to remove #{config_dir}? [y/N] "                                                                                                                                
        case STDIN.getc                                                                                                                                                                       
        when "y", "Y"                                                                                                                                                                         
          FileUtils.rm_rf config_dir                                                                                                                                                          
          puts "#{config_dir} removed.".lightred                                                                                                                                              
        else                                                                                                                                                                                  
          puts "#{config_dir} saved.".lightgreen                                                                                                                                              
        end                                                                                                                                                                                   
      end                                                                                                                                                                                     
    }
```

Reply:
This is obviously a better solution. When I get back to a desktop I can add it in, or you can.